apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -35,8 +41,20 @@ const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
                     unitType,
                     user: { connect: { id: context.authedItem.id } },
                 }
+
+                // TODO(AleX83Xpert): Think about how to prevent copying this condition :-/
+                const client = conf.NODE_ENV === 'test' || get(conf, 'ADDRESS_SERVICE_CLIENT_MODE') === 'fake'
```

Comment:
What if client can't be found in conf? Should this case been handled with an special error?

Reply:
Resident and Property models can't work without address service and without config field. If there is no config for address service, property and resident creation will fail.